tests/test_acasclient.py

```diff
@@ -2756,6 +2762,118 @@ def create_restricted_lot(self, project_code):
+
+                    # Make sure we canf ind the new lot and that the parent and salt form are same as new corp name
+                    meta_lot = self.client.get_meta_lot(wet_run_response["newLot"]["corpName"])
+                    self.assertEqual(meta_lot["lot"]["parent"]["corpName"], adopting_parent_corp_name)
+                    self.assertEqual(meta_lot["lot"]["saltForm"]["corpName"], adopting_parent_corp_name)        
+
+            except requests.HTTPError:
+                return False
+            
+            return True
+
+        # Create a bunch of users with various roles and project access
+        cmpdreg_user = self.create_and_connect_backdoor_user(acas_user=False, acas_admin=False, creg_user=True, creg_admin=False)
+        cmpdreg_user_with_restricted_project_acls = self.create_and_connect_backdoor_user(acas_user=False, acas_admin=False, creg_user=True, creg_admin=False, project_names = [project.names[PROJECT_NAME]])
+        acas_user = self.create_and_connect_backdoor_user(acas_user=True, acas_admin=False, creg_user=False, creg_admin=False)
+        acas_user_restricted_project_acls = self.create_and_connect_backdoor_user(acas_user=True, acas_admin=False, creg_user=False, creg_admin=False, project_names = [project.names[PROJECT_NAME]])
+        acas_admin = self.create_and_connect_backdoor_user(acas_user=True, acas_admin=True, creg_user=False, creg_admin=False)
+        cmpdreg_admin = self.create_and_connect_backdoor_user(acas_user=False, acas_admin=False, creg_user=True, creg_admin=True)
+
+        # Verify dry run works by doing a dry run reparent
+        # Starting state is 2 lots (1 on CMPD-0000001 and 1 on CMPD-0000002)
+        self.assertTrue(can_reparent_lot(self, self.client, "CMPD-0000001-001",  "CMPD-0000002", dry_run = True))
+        # Current state is 2 lots (1 on CMPD-0000002 and 1 on CMPD-0000002)
+
+        # Reparent a lot
+        self.assertTrue(can_reparent_lot(self, self.client, "CMPD-0000001-001",  "CMPD-0000002", dry_run = False))
+        
+        # Verify that the assay data has moved to the new lot which should be CMPD-0000002-002 dependencies
+        depdencies = self.client.get_lot_dependencies("CMPD-0000002-002")
+        self.assertIn("linkedExperiments", depdencies)
+        hasExperiment = False
+        for depExperiment in depdencies["linkedExperiments"]:
+            if depExperiment["code"] == experiment["codeName"]:
+                hasExperiment = True
+        self.assertTrue(hasExperiment)
+        
+        # Current state is 2 lots (2 on CMPD-0000002 and 0 on CMPD-0000002 (which is now deleted))
+
+        # Create a restricted lot by project to verify we can reparent it for our user auth tests
+        restricted_lot_corp_name = self.create_restricted_lot(project.code_name)
+        # Current state is 4 lots (3 on CMPD-0000002 and 1 on CMPD-0000001 (which is now re-created))
```

Comment:
It surprised me for a second that we went from 2 to 4 lots, but I guess `create_restricted_lot` really creates two lots because the SDF it loads has two entries?

Reply:
Yes, that's right.  Added a comment about this.